__tests__/features/end-to-end.steps.ts

```diff
@@ -47,6 +47,8 @@ defineFeature(loadFeature("./end-to-end.feature"), test => {
 
       await browser!.submit();
 
+      // About visit page
+
```

Comment:
```suggestion
      await browser!.submit();
```

You need to submit the page.